components/form/ErrorLabel.js

```diff
@@ -0,0 +1,20 @@
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+export function ErrorLabel({ errorClassName, name }) {
```

Comment:
This should be ErrorMessage (not ErrorLabel). We are displaying the Error Message not Label.

```suggestion

export function ErrorMessage({ errorClassName, name }) {
```

You should also rename the filename to `ErrorMessage` (I thought I dropped this in my previous comments)